[wrangler] deployment command (#6461)
cloudflare/cloudflare-docs: content/workers/wrangler/commands.md

- [`login`](#login) - Authorize Wrangler with your Cloudflare account using OAuth.
- [`logout`](#logout) - Remove Wrangler’s authorization for accessing your account.
- [`whoami`](#whoami) - Retrieve your user information and test your authentication configuration.

> - [`deployments`](#deployments) - Logs the 10 most recent deployments with 'Version ID', 'Version number','Author name', and 'Latest deploy'.

Reviewer: to be consistent with the tense used in defining the other commands, suggest using the imperative mood. and to be consistent with how `whoami` is defined, suggest changing from "log" to "retrieve". also suggest we leave the info about what is returned in the detail section for the command

```suggestion
- [`deployments`](#deployments) - Retrieve details for the 10 most recent deployments.
```

```sh
$ wrangler whoami
```

## deployments

> Logs the 10 most recent deployments with 'Version ID', 'Version number','Author name', and 'Latest deploy'

Reviewer: ```suggestion
Retrieve details for the 10 most recent deployments. Details include `Version ID`, `Version number`, `Author name`, and `Latest deploy`.
```

```sh
$ wrangler deployments
```

{{<definitions>}}

> - `--name` {{<type>}}string{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}

Reviewer: what does this flag do? we should define

Author: Done. Its allows you to specify a script name rather than Wrangler inferring it.

```sh
$ wrangler whoami
```

## deployments

> Retrieve details for the 10 most recent deployments. Details include `Version ID`, `Version number`, `Author name`, and `Latest deploy`.

Reviewer: per https://github.com/cloudflare/wrangler2/pull/2091#discussion_r1009533087 and https://github.com/cloudflare/wrangler2/pull/2091#discussion_r1009557379: 

```suggestion
Retrieve details for the 10 most recent deployments. Details include `Version ID`, `Version number`, `Author email`, `Created on`, and `Latest deploy`.
```

Author: Fixes on https://github.com/cloudflare/wrangler2/pull/2108

Author: I thought this was a different PR lol
